docs/eslint/flat-config-migration.md

```diff
@@ -0,0 +1,206 @@
+
+We can mimic the old ESLint behavior by importing each package's ESLint config into the top level and merging them together.
+
+### Monorepo packages
+
+Inside each monorepo package, create a `eslint.config.js` file. Add the config module that matches the package type. For example, a web application would use the `webApplication` config, while a component libary would use the `reactLibrary` config:
+```javascript
+import workleapPlugin from "@workleap/eslint-config";
+
+const config = workleapPlugin.configs.reactLibrary;
+
+export default config;
+```
+
+You can also set custom ignore rules:
+```javascript
+import workleapPlugin from "@workleap/eslint-config";
+
+const config = [
+    ...workleapPlugin.configs.reactLibrary,
+    ignores: ["build/"]
+];
+
+export default config;
+```
+
+### Top level
+
+Create a new `eslint.config.js` at the root of your project. Import the monorepo workspace config.
+
+In order to make it easier to scope config files to monorepo packages, we recommend using [eslint-flat-config-utils](https://github.com/antfu/eslint-flat-config-utils). The `concat` function makes it easier to join multiple configs together. Wrap your configuration objects in the `concat` function because it will automatically merege arrays.
+
+```javascript
+import { concat } from "eslint-flat-config-utils";
+import workleapPlugin from "@workleap/eslint-config";
+
+const config = concat(
+    {
+        ignores: ["node_modules/"]
+    },
+    workleapPlugin.configs.monorepoWorkspace
+);
+
+export default config;
+```
+
+Import each package's `eslint.config.js` and add them to the `concat` function. Wrap each of the package imports with the `extend` function, and provide the relative path to the root of each package. This will scope the files of that config to the given directory, including any ignores.
+
+```javascript
+import { concat } from "eslint-flat-config-utils";
+import workleapPlugin from "@workleap/eslint-config";
+import packageOneConfig from "./packages/one";
+import packageTwoConfig from "./packages/two";
+
+const config = concat(
+    {
+        ignores: ["node_modules/"]
+    },
+    workleapPlugin.configs.monorepoWorkspace,
+    extend(packageOneConfig, "packages/one/"),
+    extend(packageTwoConfig, "packages/two/"),
+);
+
+export default config;
+```
+
+With this setup, you can lint the entire project from the root, or from within each individual package directory.
+
+## Advanced Configuration
+
+We recommend using one of the "by project type" configurations for simplicity and consistency. But if you need to customize further, you can choose to combine [any individual configs](wl-web-configs/eslint/advanced-composition/). Here, we'll compose a config for a project that uses React and TypeScript. 
+
+```javascript
+import workleapPlugin from "@workleap/eslint-config";
+
+const config = [
+    ...workleapPlugin.configs.core,
+    ...workleapPlugin.configs.typescript,
+    ...workleadPlugin.configs.react
```

Comment:
Should we recommend using antfu "concat" function here?

Reply:
Yeah, I'll update this